demos/consistent-focus.html

```diff
@@ -18,212 +19,219 @@
+        <!-- Bloom effect -->
+        <calcite-block open heading="Bloom" description="Apply a neon-like glow">
+          <!-- Bloom enabled -->
+          <calcite-block-section id="bloom-effect" open text="Apply bloom effect" toggle-display="switch">
+            <!-- Intensity -->
+            <calcite-label for="bloom-intensity">Bloom intensity</calcite-label>
+            <calcite-slider id="bloom-intensity" value="1.5" min="0" max="10" step="0.5" label-ticks
+              ticks="2"></calcite-slider>
+            <!-- Radius-->
+            <calcite-label for="bloom-radius">Bloom radius</calcite-label>
+            <calcite-slider id="bloom-radius" value="0.5" min="0" max="1" step="0.5" label-ticks
+              ticks="0.25"></calcite-slider>
+            <!-- Threshold -->
+            <calcite-label for="bloom-threshold">Bloom threshold</calcite-label>
+            <calcite-slider id="bloom-threshold" value="0.1" min="0" max="0.5" step="0.1" label-ticks
+              ticks="0.25"></calcite-slider>
+          </calcite-block-section>
+        </calcite-block>
+        <!-- Drop shadow effect -->
+        <calcite-block open heading="Drop shadow" description="Apply a drop shadow">
+          <!-- Shadow enabled -->
+          <calcite-block-section id="shadow-effect" text="Apply drop shadow effect" toggle-display="switch">
             <!-- Shadow length -->
             <calcite-label for="shadow-length">Shadow length</calcite-label>
-            <calcite-slider id="shadow-length" value="3" min="0" max="4" step="1" label-ticks ticks="1"></calcite-slider>
+            <calcite-slider id="shadow-length" value="3" min="0" max="4" step="1" label-ticks
+              ticks="1"></calcite-slider>
             <!-- Shadow depth-->
             <calcite-label for="shadow-depth">Shadow depth</calcite-label>
             <calcite-slider id="shadow-depth" value="1" min="0" max="4" step="1" label-ticks ticks="1"></calcite-slider>
             <!-- Shadow outline -->
             <calcite-label for="shadow-outline">Shadow outline</calcite-label>
-            <calcite-slider id="shadow-outline" value="3" min="0" max="4" step="1" label-ticks ticks="1"></calcite-slider>
+            <calcite-slider id="shadow-outline" value="3" min="0" max="4" step="1" label-ticks
+              ticks="1"></calcite-slider>
           </calcite-block-section>
-      </calcite-block> 
-    </calcite-panel>
-  </calcite-shell-panel>
+        </calcite-block>
+      </calcite-panel>
+    </calcite-shell-panel>
 
-  <calcite-panel>
-    <arcgis-map item-id="c2a3444863f2466aaad9efa6e65063e1" id="mapEl">
-      <arcgis-legend legend-style="classic" position="bottom-right"></arcgis-legend>
-    </arcgis-map>
-  </calcite-panel>
+    <calcite-panel>
+      <arcgis-map item-id="c2a3444863f2466aaad9efa6e65063e1" id="mapEl">
+        <arcgis-legend position="bottom-right"></arcgis-legend>
```

Comment:
For UI cleanup, WDYT if we move the `arcgis-legend` into an `action` + `panel` combo?

```suggestion

```